docs/sql_service.md

```diff
@@ -0,0 +1,119 @@
+# Subgraph:SQL Service
+
+The Subgraph:SQL Service, developed by Semiotic Labs in collaboration with The Guild
+and Edge & Node, offers a secure SQL interface for querying a subgraph's entities.
+To deploy this with minimal changes to the existing indexer stack, we wrap a consumer's
```

Comment:
Oh. Sorry. Instead of "we," which is ambiguous (an indexer might ask if they have to do this), you can say something like "When a consumer sends a SQL query, they (or the Studio they use), must wrap it in a GraphQL query. This is explained in the next section."